doc/sphinx/language/core/variants.rst

```diff
@@ -6,11 +6,30 @@ Variants and the `match` construct
 Variants
 --------
 
+The :cmd:`Variant` command allows defining types by listing
+the :term:`inhabitants <inhabitant>` of the type. This includes enumerated types from
+programming languages, such as Booleans, characters or even the
```

Comment:
```suggestion
The :cmd:`Variant` command allows defining types by listing
the :term:`inhabitants <inhabitant>` of the type.  Each inhabitant is
specified by a :gdef:`constructor`.  For instance, Booleans have two
constructors: :g:`true` and :g:`false`. Types can include enumerated types from
programming languages, such as Booleans, characters or even the
```

Moved two sentences up from below and simplified wording for constructors.  (not sure why the diff shown is not more granular today)

Reply:
Yes, much better.